db/migrate/20230708053111_create_calories.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCalories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :calories do |t|
+      t.integer :user_id
+      t.string :food_name
+      t.string :meal_type
+      t.datetime :consumed_at
+      t.text :notes
+      t.integer :calories
+
+      t.timestamps
+    end
```

Comment:
One rule of thumb I used to follow to create good names when I am modeling is paying attention to the prefixes we sometimes add for some attributes. That might be a smell of an opportunity to find better names.

For example, here you've used `food_name`.
Another smell is that you have an entity named `Calories` but also have a value named `calories`. What would result in sometimes read `calorie.calories` which doesn't make much sense.

My suggestion is to name this entity `Food`.
By doing that you would have:
* `food.name` (no need for the prefix `food_` anymore.
* `food.calories` which makes much more sense.

That's just a small mindset that I believe will help you in all your career. 😃 

There is this famous quote you will see lots of people in the area agreeing with:

> There are only two hard things in Computer Science: cache invalidation and naming things.
>
> -- Phil Karlton